security/pfSense-pkg-suricata/files/usr/local/pkg/suricata/suricata_generate_yaml.php

```diff
@@ -82,16 +82,16 @@
 	"sip_servers" => "\$HOME_NET"
 );
 $addr_vars = "";
-	foreach ($suricata_servers as $alias => $avalue) {
-		if (!empty($suricatacfg["def_{$alias}"]) && is_alias($suricatacfg["def_{$alias}"])) {
-			$avalue = trim(filter_expand_alias($suricatacfg["def_{$alias}"]));
-			$avalue = preg_replace('/\s+/', ', ', trim($avalue));
-		}
-		$addr_vars .= "    " . strtoupper($alias) . ": \"{$avalue}\"\n";
+foreach ($suricata_servers as $alias => $avalue) {
+	if (!empty($suricatacfg["def_{$alias}"]) && is_alias($suricatacfg["def_{$alias}"])) {
+		$avalue = trim(filter_expand_alias($suricatacfg["def_{$alias}"]));
+		$avalue = preg_replace('/\s+/', ', ', trim($avalue));
 	}
+	$addr_vars .= "    " . strtoupper($alias) . ": \"{$avalue}\"\n";
+}
 $addr_vars = trim($addr_vars);
-if(is_array($config['system']['ssh']) && isset($config['system']['ssh']['port']))
-        $ssh_port = $config['system']['ssh']['port'];
+if(config_get_path('system/ssh/port'))
+        $ssh_port = config_get_path('system/ssh/port');
 else
         $ssh_port = "22";
```

Comment:
```suggestion
$ssh_port = config_get_path('system/ssh/port', '22');
```

Reply:
Dang it! Missed that one. Since this is already merged, I'll fix it later because it does not impact functionality.